C++/README.md

```diff
@@ -37,7 +37,7 @@ Implementation of well-known (and some rare) algorithms, in C++.
   * [Shell sort](source/sorting/shell_sort.cpp)
 * String
   * [Longest common subsequence](source/string/lcs.cpp)
-  * Searching
+  * Searching [search a pattern within string]
```

Comment:
You can change the text inside the brackets to "pattern matching". Also, please change the square brackets to parenthesis :grimacing:

Reply:
sure sure